### PR TITLE
Fixes #25151 - Properly show spool error

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Needs https://github.com/theforeman/smart_proxy_openscap/pull/65

**Comment by ares** [comment]
[test foreman_openscap] not sure if this helps, might need a rebase

**Comment by ares** [comment]
needs small rebase now again

File: app/controllers/openscap_proxies_controller.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class OpenscapProxiesController < ApplicationController
   before_action :find_proxy, :only => [:openscap_spool]
 
   def openscap_spool
-    last_error = @smart_proxy ? find_spool_error : nil
+    last_error = @smart_proxy ? @smart_proxy.statuses[:openscap].spool_status : nil
```
Comment (ares) [inline app/controllers/openscap_proxies_controller.rb]: could we rename `last_errors` to `spool_errors` or just `error` please? that we we don't have to rename in future when we add more error types :-)

File: app/views/smart_proxies/_openscap_spool.html.erb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <span>
-  <% if last_error && last_error.has_key?('timestamp') %>
+  <% if last_error && last_error['errors_count'] > 0 %>
```
Comment (ares) [inline app/views/smart_proxies/_openscap_spool.html.erb]: if last_error is renamed, it needs to be renamed everywhere here too

**Pull request merged**